_dev/src/components/product-feed-page/sync-history/sync-history.vue

```diff
@@ -81,7 +85,6 @@ export default {
             ),
             description: this.$i18n.t(
               'productFeedPage.syncSummary.syncHistory.subtitle.error',
-              {date: this.$options.filters.timeConverterToDate(this.syncInfos.jobEndedAt)},
```

Comment:
Why removing it here ?

Reply:
When I see the translation, there are no argument for date

Reply:
true! my bad :)